jaxb-ri/core/src/main/java/org/glassfish/jaxb/core/unmarshaller/DOMScanner.java

```diff
@@ -121,7 +121,7 @@ public void scan( Element e) throws SAXException {
      * @deprecated in JAXB 2.0
      *      Use {@link #scan(Element)}
      */
-    @Deprecated
+    @Deprecated(since="4.0.0-SNAPSHOT", forRemoval=true)
```

Comment:
finding the right version is a bit challenging task - one should go through the history in the original repo to find out the exact version. But `-SNAPSHOT` is definitely wrong, it must be some final version number